docs/WIP/Tutorial_Motif_only_Matrix_Profile.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def PAA(T, dsr):
</pre><p>I don't like <code>dsr</code>. Can we use <code>rate</code> or <code>downsample</code> instead? Even <code>sampling_rate</code> is okay </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>